src/gas/calculateBusyThreshold.ts

```diff
@@ -0,0 +1,28 @@
+import { BN } from 'ethereumjs-util';
```

Comment:
See https://gitlab.com/ConsenSys/codefi/products/metaswap/gas-api/-/blob/017436f628b2d5967f6e8734780a9114f9e58af9/src/server.ts#L186 and https://gitlab.com/ConsenSys/codefi/products/metaswap/gas-api/-/blob/017436f628b2d5967f6e8734780a9114f9e58af9/src/eip1559/feeHistory.ts#L152 for the original source.